Formula/parquet-tools.rb

```diff
@@ -0,0 +1,24 @@
+class ParquetTools < Formula
+  desc "ApacheParquet command-line tools and utilities"
```

Comment:
Minor nitpick: should be spelled "Apache Parquet" instead of "ApacheParquet", since that's how the project styles it.